aspnetcore/performance/rate-limit.md

```diff
@@ -0,0 +1,186 @@
+---
+title: Rate limiting middleware in ASP.NET Core
+author: rick-anderson
+description: Learn how limit requests in ASP.NET Core apps
+ms.author: riande
+ms.custom: mvc
+ms.date: 8/28/2022
+uid: performance/rate-limit
+---
+
+# Rate limiting middleware in ASP.NET Core
+
+By [Arvin Kahbazi](https://github.com/Kahbazi) and [Rick Anderson](https://twitter.com/RickAndMSFT)
+
+:::moniker range=">= aspnetcore-7.0"
+
+The [Microsoft.AspNetCore.RateLimiting](https://www.nuget.org/packages/Microsoft.AspNetCore.RateLimiting) NuGet package provides rate limiting middleware. Apps configure rate limiting policies and then attach the policies to endpoints. Apps using rate limiting should be carefully load tested before deploying. See [Testing endpoints with rate limiting](#test7) in this article for more information.
```

Comment:
This has gone into the shared framework now, so I don't think there will be a NuGet package by the time RTM ships: https://github.com/dotnet/aspnetcore/pull/43089/files#diff-bf46b41a392a3ca005eb29dd1c8fcec2f3469729856db52eaf1a4cb0fa910595